helm-chart-sources/logstream-workergroup/templates/ingress.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.ingress.enable }}
+{{- range .Values.ingress.ingress }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "logstream-workergroup.fullname" $ }}-{{ .name }}
+  annotations:
+    {{- $annotations := merge .annotations $.Values.ingress.annotations }}
+    {{- toYaml $annotations | nindent 4 }}
+spec:
+  {{- with .tls }}
+  tls:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- if or $.Values.ingress.ingressClassName .ingressClassName }}
+  ingressClassName: {{ coalesce .ingressClassName $.Values.ingress.ingressClassName }}
+  {{- end }}
+  rules:
+  {{- range .rules }}
+    - http:
+        paths:
+          {{- range .paths }}
+          - path: {{ .path }}
+            pathType: {{ .kind | default "Prefix" }}
+            backend:
+              service:
+                name: {{ include "logstream-workergroup.fullname" $ }}
+                port:
+                  number: {{ .port }}
+          {{- end }}
+    {{- if .host }}
```

Comment:
I simplified and got rid of one `else` by moving the optional host to the end

Reply:
Nice!